obspy/io/sac/sactrace.py

```diff
@@ -1092,7 +1175,7 @@ def read(cls, source, headonly=False, ascii=False, byteorder=None,
                 val = _ut._clean_str(val, strip_whitespace=False)
                 if val.startswith(native_str('-12345')):
                     val = HD.SNULL
-                hs[i] = val
+                hs[i] = val.encode('ascii', 'replace')
```

Comment:
This is the only changes I intended to make. Somehow, there appears to be a large number of changes after merging with master branch from upstream as advised by @jkmacc-LANL .